raiden-ts/src/channels/epics.ts

```diff
@@ -607,14 +613,24 @@ const makeDeposit$ = (
   approveNonce?: number,
 ): Observable<channelDeposit.failure> => {
   if (!deposit?.gt(Zero)) return EMPTY;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function stopRetryIfRejected(err: any, count: number) {
+    // 4001 is Metamask's code for rejected/denied signature prompt
+    return count >= 10 || err?.code === 4001;
```

Comment:
I usually avoid declaring single-use variables, although the overhead is minimum, it can drive up or LoC count, which is already considerable, if we start assigning every single-use constant to a variable. In this case, I can do it, but we should get used to have these values around, and comments isn't as bad, we should mind them whem changing code in the vicinity.